scripts/populate-search-fields.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+/* Since mongo does not support ES6 we can not lint with our current eslint config */
+
+String.prototype.removePunctuation = function () {
+    return this.replace(/[.,]/g,'');
+};
+
+String.prototype.removeStopWords = function () {
```

Comment:
The 'dr' substitution break some searches. 
i.e. St Andrews gets changed to 'st anew' - searching from the front end returns no results